src/gtk/artgtk.cpp

```diff
@@ -58,7 +58,18 @@ namespace
     #define ART(wxId, stockId, unused) wxId, stockId,
 #endif
 
-const wxString wxId2Gtk[] = {
+namespace {
+struct wxArtStockMapping
+{
+    wxArtID artId;
+    wxString stockId;
+};
+}
+
+wxString wxArtIDToStock(const wxArtID& id)
+{
+  static const wxArtStockMapping wxId2GtkMap[] =
```

Comment:
It's a bit confusing to call something which is an array a "map"... BTW, we could actually use `wxStringToStringHashMap` here.